packages/vite/src/generators/configuration/schema.d.ts

```diff
@@ -1,5 +1,5 @@
 export interface ViteConfigurationGeneratorSchema {
-  uiFramework: 'react' | 'none';
+  uiFramework: 'angular' | 'react' | 'none';
```

Comment:
Are we using the `angular` value to determine anything? Could Angular's call to this generator use `none`?

Reply:
Good catch! I'll remove it.
IIRC, I think that at some point, the logic of adding the `src/test-setup.ts` was in this generator then I moved it up.

Reply:
Done!